src/common/tiles/MovieTile/styled.js

```diff
@@ -1,5 +1,5 @@
 import styled from "styled-components";
-import { Ratings } from "./Ratings/index.js";
+import { Ratings } from "./Ratings";
```

Comment:
Nie wiem czy to dobry pomysl, zeby w ten sposob stylowac caly komponent powiem szczerze, ze dla mnie bylo to troche malo czytelne, chyba latwiej by bylo wrzucic cale Ratings do MovieTile index.js a stylowanie komponentow do styled.js rowniez w MovieTile ale to tylko moja opinia i mysle, ze tak jak jest tez jest w porzadku i moze niech lepiej tak zostanie, poprostu pisze na przyszlosc co uwazam w tym temacie(mozemy to przedyskutowac na callu) :)